app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/repository/di/AuthenticatorBridgeModule.kt

```diff
@@ -42,7 +43,12 @@ object AuthenticatorBridgeModule {
     ): AuthenticatorBridgeManager =
         if (featureFlagManager.getFeatureFlag(LocalFeatureFlag.PasswordManagerSync)) {
             factory.getAuthenticatorBridgeManager(
-                connectionType = AuthenticatorBridgeConnectionType.DEV,
+                connectionType = when (BuildConfig.BUILD_TYPE) {
+                    "release" -> AuthenticatorBridgeConnectionType.RELEASE
+                    "debug" -> AuthenticatorBridgeConnectionType.DEV
```

Comment:
This definitely feels a bit janky to me. I checked out the main app to see where we were making checks on `BUILD_TYPE` and didn't find much. 

Should I go as far as having different Module per source set?

Reply:
I agree it feels a little too janky. I fully expect us to eventually provide F-Droid builds, so I think a more elegant approach would be better. If you like having a separate source file, I'm cool with that. We could also introduce a `buildConfigField` typed to the `AuthenticatorBriddeConnectionType` enum. Either should be more scalable than checking the build type.

Reply:
Thanks for the feedback! After a night of sleep, I definitely agree :) 

I like the `buildConfigField` idea... I'll look into that when I circle back.